scripts/add-perms-to-lambda.sh

```diff
@@ -5,6 +5,15 @@ set -o errexit -o pipefail -o nounset
 
 dist_path="$(pwd)/$1"
 
-cd "${dist_path}/nrlf_permissions" && zip -q -r "${dist_path}/nrlf_permissions.zip" . && rm -rf "${dist_path}/nrlf_permissions"
+mkdir -p "${dist_path}/nrlf_permissions-layer/python/nrlf_permissions"
+cp -r "${dist_path}/nrlf_permissions"/* "${dist_path}/nrlf_permissions-layer/python/nrlf_permissions"
+
+cd "${dist_path}/nrlf_permissions-layer/" && \
+    rm -f ${dist_path}/nrlf_permissions.zip && \
+    zip -q -r "${dist_path}/nrlf_permissions.zip" . \
+        -x **tests**\* **scripts**\* **__pycache__**\*
+
+rm -rf "${dist_path}/nrlf_permissions-layer"
+rm -rf "${dist_path}/nrlf_permissions"
```

Comment:
As this script doesn't create this directory, it might be better to not delete it in this script

Not a biggie so happy to leave